plugins/innogrit/typedef.h

```diff
@@ -0,0 +1,103 @@
+
+typedef void VOID;

```

Comment:
Please don't add new and redefined typedefs. We'd prefer common coding style across the whole project instead of every vendor extension introducing their special dialect.